src/app/register/page.tsx

```diff
@@ -0,0 +1,124 @@
+"use client";
+import { useState } from "react";
+import {
+  Container,
+  Typography,
+  TextField,
+  Button,
+  Snackbar,
+} from "@mui/material";
+import { useNavigate } from "react-router-dom";
```

Comment:
```suggestion
import { useNavigate } from "react-router-dom";
```

요거 삭제가 안 되었네요...! 확인 부탁합니다~(불필요한 import)